diagnostics/physics/sst_eval.py

```diff
@@ -17,23 +17,25 @@
 from plot_common import annotate_skill, autoextend_colorbar, corners, get_map_norm, open_var
 
 
-def plot_sst_eval(pp_root):
-    model = open_var(pp_root, 'ocean_monthly', 'tos')
-    model_grid = xarray.open_dataset('../data/geography/ocean_static.nc')
-    target_grid = model_grid[['geolon', 'geolat', 'geolon_c', 'geolat_c']].rename({'geolon': 'lon', 'geolat': 'lat', 'geolon_c': 'lon_b', 'geolat_c': 'lat_b'})
+def plot_sst_eval(pp_root, config):
+    model = open_var(pp_root, config['domain'], 'tos')
+    model_grid = xarray.open_dataset( config['model_grid'] )
+    target_grid = model_grid[config['rename_vars']].rename(config['rename_map'])
```

Comment:
It should be possible to do something like `model_grid[config['rename_map'].keys()].rename(etc)` or `model_grid[list(config['rename_map'].keys())].rename(etc)` so that you don't need the extra `rename_vars` in the yaml